src/controller/python/templates/MatterIDL.zapt

```diff
@@ -0,0 +1,125 @@
+// This IDL was generated automatically by ZAP.
```

Comment:
This files looks duplicated. there should be no source code copy and paste for this.

Reply:
No this file is not copy&pasted.

Unlike the MatterIDL.zapt under app/templates, this file does not rely on the .zap files (thus it won't emit endpoint config)

Reply:
It still looks extreamly similar to MatterIDL.zapt, hence my copy&paste comment. Are you saying you wrote this from scratch by hand?

We should be sharing code as much as possible, if this involves a new template fine.

Could we just ignore the endpoint config for python and use the other template?